CHANGELOG.md

```diff
@@ -22,6 +22,10 @@ The minor version will be incremented upon a breaking change and the patch versi
 
 ### Breaking
 
+## Next release
+
+- cli: add filename to 'Unable to read keypair file' errors. ([#2932](https://github.com/coral-xyz/anchor/pull/2932))
+
```

Comment:
Please follow the existing template similar to https://github.com/coral-xyz/anchor/pull/2929/commits/0acc8aaae2f70e6884df511b12d46ad18dab6d89.

Reply:
I did look at the existing entry, I just missed the `## [Unreleased]` heading.